<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pointSelectDlg</class>
 <widget class="QDialog" name="pointSelectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>219</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>219</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>282</width>
    <height>219</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>测点关联定义</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../mainwindow.qrc">
    <normaloff>:/image/config.png</normaloff>:/image/config.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>厂站</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="stationCombo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>间隔</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="groupCombo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>类型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeCombo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>关联测点</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="relatedPointCombo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="undoBtn">
       <property name="text">
        <string>撤销关联</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="noBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>groupCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pointSelectDlg</receiver>
   <slot>currentGroupChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>relatedPointCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pointSelectDlg</receiver>
   <slot>currentPointChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>155</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>undoBtn</sender>
   <signal>clicked()</signal>
   <receiver>pointSelectDlg</receiver>
   <slot>dropRelation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>pointSelectDlg</receiver>
   <slot>selectRelation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noBtn</sender>
   <signal>clicked()</signal>
   <receiver>pointSelectDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pointSelectDlg</receiver>
   <slot>currentTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>currentGroupChanged(int)</slot>
  <slot>currentPointChanged(int)</slot>
  <slot>dropRelation()</slot>
  <slot>selectRelation()</slot>
  <slot>cancelRelation()</slot>
  <slot>currentTypeChanged(int)</slot>
 </slots>
</ui>
